example/package.json

```diff
@@ -3,6 +3,7 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@textkernel/oneui": "^0.1.0",
```

Comment:
@ribeiroguilherme would it make more sense to  use `*` since the example is always part of the same (latest) release? Otherwise we'd have to keep updating this I suppose

Reply:
Done